FLEDGE.md

```diff
@@ -90,9 +90,19 @@ const myGroup = {
   'biddingLogicUrl': ...,
   'dailyUpdateUrl': ...,
   'trustedBiddingSignalsUrl': ...,
-  'trustedBiddingSignalsKeys': ['key1', 'key2'],
+  'trustedBiddingSignalsKeys': {
+      'productKeys': ['key1', 'key2'],
+      'campaignKeys': ['key3', 'key4'],
+      ...
+  },
   'userBiddingSignals': {...},
-  'ads': [shoesAd1, shoesAd2, shoesAd3],
+  'adComponents': {
```

Comment:
> Just to make sure I understand, it seems like this might well not be used as components, right? Interest Group owners might want to use this instead to organize their ads, to make use of your shallow merge idea, even if they aren't composing any particular creative out of individual pieces. 

Yes. The idea behind that name was that an ad with no products could be seen as an ad composed of a single component. In that sense, "component" could mean both "ad" or "product".

> This makes me wonder if a name other than adComponents would better communicate the idea, though I don't have a good counter-suggestion offhand.

- We could stay with just `ads`, and accept this field may also include products.
- Perhaps the cleanest approach would be to have two fields, `ads`, and `adComponents`? (Or `ads` and `products`?)

> For people who are content to lump all their ads together, they'd need to pick an arbitrary key, right? 'adComponents': {'all': [shoesAd1, shoesAd2, shoesAd3]}

Yes.